src/status_im/ui/screens/discover/styles.cljs

```diff
@@ -61,37 +70,55 @@
    :padding-right  9})
 
 (def separator
-  {:background-color "rgb(200, 199, 204)"
-   :height           0.5})
+  {:background-color "rgb(238, 241, 245)"
```

Comment:
Is this a global palette thing? If so it should probably be named in variable along with other colors. If not, i.e. just experimental for now, we can leave it alone for now

Reply:
The global palette colors were not the same as the ones I color-picked from designs at the time. I think that might actually be an invisionapp issue. Can you confirm @denis-sharypin ?
In any case, I kind of handwaved the issue until the pixel-good once over part.

Reply:
Ok that's fair, let's leave for later PR